File: Data Structures & Algorithms/top-k-elements-in-list/submission-5.cs
public class Solution {
    public int[] TopKFrequent(int[] nums, int k) {

        
            List<int> res = new List<int>();
            Dictionary<int, int> dic = new Dictionary<int, int>();
            foreach (int num in nums)
            {
                if (!dic.ContainsKey(num))
                {
                    dic.Add(num, 1);
                }
                else
                {
                    dic[num] = dic[num] + 1;
                }


            }

             var newdic = dic.OrderByDescending(x => x.Value);


            foreach (var item in newdic)
            {
                res.Add(item.Key);
            }

            return res.Take(k).ToArray();
        
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-5.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int res=0,curCount=0;
        for(int n : nums){
            if(n==1)
                curCount++;
            else{
                res = curCount > res ? curCount : res ;
                curCount = 0;
            }
                
        }
        return curCount > res ? curCount : res ;
    }
};